src/main/webapp/assets/js/authentication.js

```diff
@@ -25,16 +25,29 @@ async function isLoggedin() {
  * and a link to log out
  */
 function navBarUserLoggedIn(userEmail, logoutUrl, isAdmin) {
-    if (isAdmin === "true") {
-        return `
-        <a href="add.html" class="nav-link">Add New Business</a>
-        <a href="#" class="nav-link">${userEmail}</a>
-        <a href="${logoutUrl}" class="nav-link">Log Out</a>`;
+  const queryString = window.location.search;
+  const urlParams = new URLSearchParams(queryString);
+  const businessId = urlParams.get("businessID");
+
+  if (isAdmin === "true") {
+    if (window.location.pathname.includes("product.html"))
+    {
+      return `
+      <a href="edit.html?businessID=${businessId}" class="nav-link">Edit This Business</a>
```

Comment:
Design wise, I would recommend against having Edit This Business in the navbar because the navbar should contain "global" things. I would suggest to put it in a button on product page portion itself. The Add New Business should still remain on the navbar because an admin should be able to add a new business anytime without having to navigate to the homepage.

Reply:
Moved to the product page itself!